CMakeLists.txt

```diff
@@ -53,11 +55,12 @@ ADD_SUBDIRECTORY(tests)
 ENABLE_TESTING()
 ADD_TEST(test_pfx tests/test_pfx)
 ADD_TEST(test_lpfst tests/test_lpfst)
-ADD_TEST(test_pfx_locks tests/test_pfx_locks)
+#ADD_TEST(test_pfx_locks tests/test_pfx_locks)
```

Comment:
Agree. However, this significant change should be part of the commit message ;).